lib/main.dart

```diff
@@ -1,5 +1,5 @@
 import 'package:flutter/material.dart';
-import 'package:smarn/Home.dart';
+import 'package:smarn/home.dart'; // Ensure you have this file
 
 void main() {
   runApp(const MyApp());
```

Comment:
Why does your main function not contain the firebase configuration I added before, your remote ui-ux branch doesn't seem to be updated from develop, check the main file in ui-ux and the main file in develop branch here in github and you'll see the difference.